apps/src/templates/Dialog.jsx

```diff
@@ -173,7 +194,7 @@ const Dialog = React.createClass({
       children.push(<Footer key="footer">{this.props.footer}</Footer>);
     }
     return (
-      <BaseDialog {...this.props}>
+      <BaseDialog {...this.props} ref="baseDialog" handleKeyDown={this.handleKeyDown}>
```

Comment:
string based refs are considered "legacy" according to the react docs. Might want to use a function instead: https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute